packages/dds/matrix/src/test/utils.ts

```diff
@@ -222,3 +224,27 @@ export function insertFragmented(
 
 	return matrix;
 }
+
+/**
+ * Creates a local matrix with the specified size and initial value.
+ */
+export function createLocalMatrix(
+	id: string,
+	size: number,
+	initialValue?: string,
+): ISharedMatrix & IChannel {
+	const matrix = matrixFactory.create(new MockFluidDataStoreRuntime(), id);
+	for (let i = 0; i < size; i++) {
+		matrix.insertRows(i, 1);
+		matrix.insertCols(i, 1);
+	}
+
+	if (initialValue === undefined) {
```

Comment:
I think this check is inverted
```suggestion
	if (initialValue !== undefined) {
```